index.bs

```diff
@@ -115,7 +118,16 @@ spec: webidl
       The UA may collect information about a user's intentions in any way its
       authors believe is appropriate. This information can come from explicit
       user action, aggregate behavior of both the relevant user and other users,
-      or other sources this specification hasn't anticipated.
+      or <a>implicit signals</a> this specification hasn't anticipated.
+      <div class="note">
+        The <dfn>implicit signals</dfn> could be, for example, the
+        <a>install</a> status of a web application, frequency and recency of
```

Comment:
"... application *or* frequency and recency ...", I think.

Or to reword the whole sentence:
> For example, a UA could use the user's frequency and recency of visits to the application, or the fact they've [=installed=] it, as <dfn>implicit signals</dfn>, since a user that ...

Reply:
Fixed.